src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+html {
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+input {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  outline: none;
+  background-color: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  text-shadow: 2px 10px 20px gray;
+  box-shadow: none;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+}
+
+.big__search {
+  padding: 30px 62px;
+  line-height: 22px;
+  height: 70px;
+  font-size: 16px;
+}
+
+.big__search::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.small__search {
```

Comment:
My suggestion is to add a common class for input so you can gather common styles under this class and add a modifier to style smaller search. Check my example for styling placeholder this way:
```html
//html file

//big input
      <input
          class="search"
          type="search"
          name="bigSearch"
          id="bigSearch"
          placeholder='Try "Los Angeles"'
          data-qa="keypress"
      />

//small input with a modifier class
      <input
          class="search search--small"
          type="search"
          name="smallSearch"
          id="smallSearch"
          placeholder='Try "Los Angeles"'
      />


//css file
.search::placeholder {
  font-size: 16px;
  line-height: 20px;
  color: #3d4e61;
}

.search--small::placeholder {
  font-size: 14px;
}

.search:focus::placeholder {
  font-weight: normal;
  text-shadow: none;
}

```

Try to make reusable styles where you can :)